### ISSUE TITLE
babel-plugin-transform-decorators-legacy requirement

### ISSUE THREAD

**Issue by tauren** [jasonslyvia/react-lazyload#72]
I have a project based on [react-boilerplate](https://github.com/react-boilerplate/react-boilerplate) that uses webpack with babel to build for production. When I include `react-lazyload` as a dependency, I get an error when building:

```
ERROR in ./~/react-lazyload/lib/index.js
Module build failed: ReferenceError: Unknown plugin "transform-decorators-legacy" specified in "/Users/tauren/repos/react-boilerplate/node_modules/react-lazyload/.babelrc" at 0, attempted to resolve relative to "/Users/tauren/repos/react-boilerplate/node_modules/react-lazyload"
```

I've resolved the issue by adding `babel-plugin-transform-decorators-legacy` as a dependency of my project and adding it to my babel plugins configuration.  But I would think this shouldn't be necessary since react-lazyload provides already transpiled code. 

I'm guessing that when babel runs in my project, it picks up the `.babelrc` in `react-lazyload`. Perhaps `.babelrc` should be added to the `.npmignore` file so that it isn't published to NPM?

**Comment by jasonslyvia** [comment]
As a matter of fact `react-lazyload` should not be taken care of by `babel`, not sure why it is raising this error. 

Although adding `.babelrc` to `.npmignore` is surely achievable.

**Comment by jasonslyvia** [comment]
Since it's unbreaking change, I just bumped a patch version to `2.2.3` which ignores `.babelrc` and some other stuff in `.npmignore`.

**Comment by tauren** [comment]
Thanks for jumping on this. However, it looks like you've got a misspelling in your `.gitignore`. It shouldn't be `.babalrc`.

**Comment by jasonslyvia** [comment]
Oh, that's unfortunate! I released another version(`2.2.4`) with corrected spelling.

**Comment by tauren** [comment]
Thanks!
